src/plugin-support.c.in

```diff
@@ -0,0 +1,39 @@
+/*
+Plugin Name
+Copyright (C) <Year> <Developer> <Email Address>
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with this program. If not, see <https://www.gnu.org/licenses/>
+*/
+
+#include <plugin-support.h>
+
+const char *PLUGIN_VERSION = "@CMAKE_PROJECT_VERSION@";
+const char *PLUGIN_NAME = "@CMAKE_PROJECT_NAME@";
```

Comment:
I think in obs-studio we wanted to make these `static const char *`. Is there a practical difference here?

Reply:
Thanks. I always forget the particulars.